packages/strapi-parts/src/Combobox/utils.js

```diff
@@ -0,0 +1,126 @@
+export const Keys = {
```

Comment:
There's already a list of Keyboard keys in the utils folder, could you please rely on these? 🙏🏻